include/libopencm3/pac55xx/ccs.h

```diff
@@ -0,0 +1,154 @@
+/** @file
+ *
+ *  Clock Control and System Defines for the Qorvo PAC55xx series of microcontrollers.
+ *
+ *  @defgroup system_defines Clock Config and System Defines.
+ *  @ingroup PAC55xx
+ *  @author Brian Viele <vielster@allocor.tech>
+ *  LGPL License Terms @ref lgpl_license
+ *  @date 1 Dec 2019
+ *
+ *  Definitions in this file come from the PAC55XX Family User Guide Rev 1.21
+ *  by Active-Semi dated August 26, 2019.
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef INCLUDE_LIBOPENCM3_PAC55XX_CCS_H_
+#define INCLUDE_LIBOPENCM3_PAC55XX_CCS_H_
+
+#include <libopencm3/cm3/common.h>
+
+/* clang-format off */
```

Comment:
what? that can't be right

Reply:
So, without this, it will collapse all macro spacing (meaning no macro alignment). I'll delete these now that everything is properly formatted.